src/powershell/Add-ExportScopeConfiguration.psm1

```diff
@@ -0,0 +1,137 @@
+<#
+.SYNOPSIS
+Adds an export scope configuration to the specified FinOps Hub.
+
+.PARAMETER HubName
+The name of the FinOps Hub.
+Assumes HubName = ResourceGroupName.
+
+.PARAMETER Scope
+The Export Scope to add to the FinOps Hub configuration.
+
+.PARAMETER TenantId
+The Azure AD Tenant linked to the export scope.
+
+.PARAMETER Cloud
+The Azure Cloud the export scope belongs to.
+
+.EXAMPLE
+Add-ExportScopeConfiguration -HubName FinOps-Hub -TenantId 00000000-0000-0000-0000-000000000000 -Cloud AzureCloud -Scope "/providers/Microsoft.Billing/billingAccounts/1234567"
+
+Adds an export scope configuration to the specified FinOps Hub.
+#>
+Function Add-ExportScopeConfiguration {
+    [CmdletBinding()]
+    param (
+        [Parameter()]
+        [string]
+        [ValidateNotNullOrEmpty()]
+        $HubName,    
+        [Parameter()]
+        [string]
+        [ValidateNotNullOrEmpty()]
+        [ValidateSet("AzureCloud", "AzureUSGovernment")]
+        $Cloud,
+        [Parameter()]
+        [String]
+        [ValidateNotNullOrEmpty()]
+        $Scope,
+        [Parameter()]
+        [String]
+        [ValidateNotNullOrEmpty()]
+        $TenantId
+        
+    )
+    $ErrorActionPreference = 'Stop'
+    [string]$operation = 'create'
+
+    # Main
+    Write-Output ''
+    Write-Output ("{0}    Starting" -f (Get-Date))
+
+    if (!$Scope.StartsWith('/')) {
+        $Scope = '/' + $Scope
+    }
+
+    if ($Scope.EndsWith('/')) {
+        $Scope = $Scope.Substring(0, $Scope.Length - 1)
+    }
+
+    Write-Output ("{0}    Export Scope to add: {1}" -f (Get-Date), $Scope)
+    Write-Output ("{0}    tenantId for scope: {1}" -f (Get-Date), $TenantId)
+
+    $resourceGroup = Get-AzResourceGroup -Name $HubName -ErrorAction SilentlyContinue
+    if ($null -eq $resourceGroup) {
+        Write-Output ("{0}    FinOps Hub {1} Not Found" -f (Get-Date), $HubName)
+        Throw ("FinOps Hub {0} Not Found" -f $HubName)
+    }
+
+    Write-Output ("{0}    FinOps Hub {1} Found" -f (Get-Date), $HubName)
+
+    $storageAccount = Get-AzStorageAccount -ResourceGroupName $HubName -ErrorAction SilentlyContinue
+    if ($null -eq $storageAccount) {
+        Write-Output ("{0}    Storage Account Not Found" -f (Get-Date))
+        Throw ("Storage Account Not Found")
+    }
+
+    if ($storageAccount.Count -gt 1) {
```

Comment:
Instead of failing, can we see which one has `config/settings.json`? We can also use the naming pattern to isolate the right one.